docs/sphinx_docs/Plugins/index.md

```diff
@@ -13,7 +13,9 @@ The `fm-simplex` inequality plugin comes built-in with Alt-Ergo and no further
 installation is required. It is distributed under the same licensing conditions
 as Alt-Ergo. It can be used as follows:
 
-    $ alt-ergo --inequalities-plugin fm-simplex [other-options] file.<ext>
+```sh
```

Comment:
`sh` is the lexer for (ba)sh scripts; `console` should be used for interactive sessions.